server/database/fixtures/PictureSeeder.js

```diff
@@ -13,28 +13,19 @@ class PictureSeeder extends AbstractSeeder {
 
   run() {
     const images = [
-      {
-        image: "/image_1.jpg",
-      },
-      {
-        image: "/image_2.jpg",
-      },
-      {
-        image: "/image_3.jpg",
-      },
-      {
-        image: "/image_4.jpg",
-      },
-      {
-        image: "/image_5.jpg",
-      },
+      { image: "/image_1.jpg", },
+      { image: "/image_2.jpg", },
+      { image: "/image_3.jpg", },
+      { image: "/image_4.jpg", },
+      { image: "/image_5.jpg", },
     ];
 
     images.forEach((image, index) => {
       const imageWithRefName = {
         ...image,
         user_id: this.getRef(`user_${index}`).insertId,
         art_id: this.getRef(`art_${index}`).insertId,
+        id: index + 1,
```

Comment:
what is the "id: index+1" for? I think we should keep the version that is on dev

Reply:
This too is not coming from me, so maybe I got these when I pulled dev, I dunno, it does not make sense

Reply:
it is not our version on dev and dev is working, so the line "id: index + 1"  should be removed please
(I think it was added when you had add other urls in this file)

Reply:
ok line gone I will push again